java-buildpack/bin/build

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+echo "---> Java buildpack"
+
+set -eo pipefail
+
+env_dir=$1/env
+cache_dir=$2
+launch_dir=$3
+
+# Load user-provided build-time environment variables
+if compgen -G "$env_dir/*" > /dev/null; then
+  for var in "$env_dir"/*; do
+    declare "$(basename "$var")=$(<"$var")"
+  done
+fi
+
+echo "---> Installing JDK"
+
+if [[ ! -f $launch_dir/jdk.toml ]]; then
+  mkdir -p $launch_dir/jdk
+  jdk_url="https://cdn.azul.com/zulu/bin/zulu8.28.0.1-jdk8.0.163-linux_x64.tar.gz"
+  curl -sfL "$jdk_url" | tar pxz -C $launch_dir/jdk --strip-components=1
+  mkdir -p $launch_dir/jdk/.profile.d
+  cat << EOF > $launch_dir/jdk/.profile.d/jdk.sh
+export JAVA_HOME=$launch_dir/jdk
+export PATH=\$JAVA_HOME/bin:\$PATH
+export LD_LIBRARY_PATH="\$JAVA_HOME/jre/lib/amd64/server:\$LD_LIBRARY_PATH"
+EOF
+  echo "version = \"1.8.0_163\"" > $launch_dir/jdk.toml
+
+  # TODO put the jdk in the cache
+  # TODO write to $cache_dir/jdk/env
+fi
+source $launch_dir/jdk/.profile.d/jdk.sh
+
+echo "---> Installing Maven"
+
+mkdir -p $cache_dir/maven/.m2
```

Comment:
Granted that we don't do builds, but if we did I'd absolutely require the use of `mvnw` and `gradlew` to avoid any conflicts here.

Reply:
for sure, in the next interation. i plan to support both (although i might kill the default mvn install).